frame/nomination-pools/src/tests.rs

```diff
@@ -5292,3 +5368,1231 @@ mod reward_counter_precision {
 			});
 	}
 }
+
+mod commission {
+	use super::*;
+
+	#[test]
+	fn set_commission_works() {
+		ExtBuilder::default().build_and_execute(|| {
+			let pool_id = 1;
+			let root = 900;
```

Comment:
nit: `root` is a bit misleading, perhaps `pool_owner` or `pool_root` would be better.